src/main/java/it/pagopa/afm/marketplacebe/controller/PspController.java

```diff
@@ -148,8 +147,8 @@ public ResponseEntity<BundleCreditorInstitutionResource> getBundleCreditorInstit
             @Size(max = 35) @Parameter(description = "PSP identifier", required = true) @PathVariable("idpsp") String idPsp,
             @Parameter(description = "Bundle identifier", required = true) @PathVariable("idbundle") String idBundle,
             @Parameter(description = "CI fiscal code") @RequestParam(required = false) @Valid String ciFiscalCode,
-            @Positive @Parameter(description = "Number of items for page") @RequestParam(required = false, defaultValue = "50") @Max(100) Integer limit,
-            @PositiveOrZero @Parameter(description = "Page number. Page number value starts from 0") @RequestParam(required = false, defaultValue = "0") @Max(10000) Integer page) {
+            @Parameter(description = "Number of items for page") @RequestParam(required = false, defaultValue = "50") @Positive Integer limit,
```

Comment:
why did you remove the `max`?

Reply:
this API will be invoked by the cron-job on BO, the `max` creates a constraint on how mutch items we can process at time. I removed it to align the API with other similar API like get offers or get requests and to simplify the configuration of the cron-job on BO.
Otherwise we can reintroduce the constraint and increase it to `200`

Reply:
Yes, put a max value (200 is good) to avoid resources-consuming issues